exercises/reverse-string/src/main/java/ReverseString.java

```diff
@@ -1,6 +1,6 @@
 class ReverseString {
   
-    String reverse() {
+    String reverse(String input) {
```

Comment:
Nice catch! Let's go with `String inputString` to match the reference solution :)